reconcile/utils/jjb_client.py

```diff
@@ -239,11 +239,11 @@ def update(self):
                                         check=True,
                                         stdout=PIPE,
                                         stderr=STDOUT)
-                out_str = result.output.decode("utf-8")
+                out_str = result.stdout.decode("utf-8")
                 if re.search("updated: [1-9]", out_str):
                     logging.info(out_str)
             except CalledProcessError as ex:
-                msg = ex.output.decode("utf-8")
+                msg = ex.stdout.decode("utf-8")
```

Comment:
I think it's worth a follow-up here because there could be useful information in either `stdout` or `stderr`. That being said, `output` in `CalledProcessError` is the same as `stdout`, so it doesn't look like anything is being changed here (but you are fixing the line above).

Reply:
stderr is sent to STDOUT in subprocess.run() so I assume - perhaps a bit naively - that stderr isn't magically back in STDERR in CalledProcessError. I could not find a mention about it in the docs